cmd/kubelet/app/server.go

```diff
@@ -204,6 +205,9 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	// Flags intended for testing, not recommended used in production environments.
 	fs.BoolVar(&s.ReallyCrashForTesting, "really_crash_for_testing", s.ReallyCrashForTesting, "If true, when panics occur crash. Intended for testing.")
 	fs.Float64Var(&s.ChaosChance, "chaos_chance", s.ChaosChance, "If > 0.0, introduce random client errors and latency. Intended for testing. [default=0.0]")
+
+	// HACK: are you containerized?
+	fs.BoolVar(&s.Containerized, "containerized", s.Containerized, "Indicates whether kubelet is running in a container")
```

Comment:
Would it be better to check for /.dockerenv or /.dockerinit instead of having to make the user remember on the command line?